packages/client/src/components/CreatePost.js

```diff
@@ -1,96 +1,117 @@
 import { useState } from "react";
-import ButtonDiv from "./ButtonDiv";
-import Button from "./Button";
-import InputDiv from "./InputDiv";
-import Input from "./Input";
-import InputLabel from "./InputLabel";
-import TextArea from "./TextArea";
-import Checkbox from "./Checkbox";
-import Form from "./Form";
-import FormHeader from "./FormHeader";
+import { Redirect } from "react-router-dom";
+import ButtonDiv from "./styled/ButtonDiv";
+import Button from "./styled/Button";
+import InputDiv from "./styled/InputDiv";
+import Input from "./styled/Input";
+import InputLabel from "./styled/InputLabel";
+import TextArea from "./styled/TextArea";
+import Form from "./styled/Form";
+import FormHeader from "./styled/FormHeader";
+import Container from "./styled/Container";
+import TextLink from "./styled/TextLink";
+
+// Post Model looks like:
+// title: {
+//   type: String,
+//   required: true,
+// },
+// body: {
+//   type: String,
+//   required: true,
+// },
+// isDraft: {
+//   type: Boolean,
+//   required: true,
+// },
+// publishedAt: {
+//   type: Date,
+// },
 
-// Data looks like this (for backend)
-// console.log({
-//     title:'string',
-//     body:'string',
-//     isDraft:boolean
-// })
 const backend = process.env.REACT_APP_BACKEND;
-const endpoint = `${backend}/api/createPost`;
+const endpoint = `${backend}/api/create-post`;
 function CreatePost() {
-  const [title, setTitle] = useState("");
-  const [body, setBody] = useState("");
-  const [isDraft, setIsDraft] = useState(true);
+  const [post, setPost] = useState({
+    title: "",
+    body: "",
+    isDraft: true,
+  });
+
+  const [redirect, setRedirect] = useState(false);
+
+  const createPostObject = (e) => {
```

Comment:
I'm not sure `createPostObject` is the right prasing about this, I might just do `handlePostInput`